packages/core/test/util/tokenTransferInspector.test.ts

```diff
@@ -1049,5 +1073,75 @@ describe('txInspector', () => {
         ])
       );
     });
+
+    it.only('has no resolved fromAddresses on timeout', async () => {
```

Comment:
```suggestion
    it('has no resolved fromAddresses on timeout', async () => {
```

Reply:
Apologies, it should be fixed by 87baffa8fd705826f0073bec5d318204a87344e5 :)